src/Symfony/Component/HttpFoundation/Session/Storage/Handler/RedisSessionHandler.php

```diff
@@ -62,15 +62,15 @@ public function __construct($redis, array $options = [])
     /**
      * {@inheritdoc}
      */
-    protected function doRead($sessionId): string
```

Comment:
Why do you remove existing return types?

Reply:
I think this got removed by accident when i reverted the other return types :)